src/celeritas/em/model/GammaNuclearModel.cc

```diff
@@ -98,5 +112,64 @@ void GammaNuclearModel::step(CoreParams const&, CoreStateDevice&) const
 }
 #endif
 
+//---------------------------------------------------------------------------//
+/*!
+ * Build CHIPS gamma-nuclear element cross sections using G4GammaNuclearXS
+ * above the upper energy limit of G4PARTICLEXS/gamma (IAEA) data. The cross
+ * sections are derived from the parameterization developed by M. V. Kossov
+ * (CERN/ITEP Moscow) in the high energy region (106 MeV < E < 50 GeV) and
+ * from a Reggeon-based parameterization in the ultra-high-energy region
+ * (E > 50 GeV). G4GammaNuclearXS uses CHIPS (G4PhotoNuclearCrossSection)
+ * above 150 MeV and performs linear interpolation between the upper energy
+ * limit of G4PARTICLEXS/gamma (IAEA) data and 150 MeV.
```

Comment:
Our user/methods manual pulls in documentation only from the class documentation (in the `.hh` file): should we move this to the class docs where it's more visible? Does the Kossov citation have a reference?

Reply:
I move the description to the header and add a citation for the CHIPS gamma-nuclear cross section parameterization in the doc  - assume that I do not need to add the paper itself to zotero"

Reply:
Zotero exports the bibliography entries into our references, so we do need it there, so I've added it myself. Thanks!